mscore/events.cpp

```diff
@@ -612,9 +614,9 @@ void ScoreView::mousePressEvent(QMouseEvent* ev)
         } else {
             editData.element->mousePress(editData);
             _score->update();
-            if (_score != _drawingScore) { // only run for multi-movement scores
-                _drawingScore->update(); // seems to fix a bug where the individual scores wouldn;t be redrawn before doing something after changing from the album-mode score
-                _drawingScore->doLayout();
+            if (_score != m_drawingScore) { // only run for multi-movement scores
+                m_drawingScore->update(); // seems to fix a bug where the individual scores wouldn;t be redrawn before doing something after changing from the album-mode score
+                m_drawingScore->doLayout();
```

Comment:
You're using this code in three places. Could it be implemented in a separate function?